src/Pages/BookDetails/Aside.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+function Aside() {
+  const [save, setSave] = useState(false);
+
+  return (
+    <AsideContainer>
+      <Container>
+        <ul>
+          <Menu save={save} onClick={() => setSave(!save)}>
+            <img
+              alt=""
+              src={
+                save
+                  ? "https://i.ibb.co/wSDskYv/shhhh.png"
+                  : "https://cdn4.iconfinder.com/data/icons/app-custom-ui-1/48/Download-256.png"
+              }
+            />
+            <button>내 서재에 담기</button>
+          </Menu>
+          <Menu>
+            <img
+              alt=""
+              src="https://cdn4.iconfinder.com/data/icons/basic-user-interface-elements/700/share-sharing-socialmedia-social-network-exchange-256.png"
+            />
+            <button>공유하기</button>
+          </Menu>
+        </ul>
+        <ReadNow>바로 읽기</ReadNow>
+      </Container>
+    </AsideContainer>
+  );
+}
+
+export default Aside;
+
+const AsideContainer = styled.div`
+  width: 280px;
+  position: relative;
+`;
+
+const Container = styled.aside`
+  position: fixed;
+  top: 64px;
+  width: 280px;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: column;
+  border-left: 1px solid rgb(223, 223, 223);
+  border-right: 1px solid rgb(223, 223, 223);
+`;
+
+const Menu = styled.li`
+  height: 47px;
+  width: 256px;
+  background-color: transparent;
+  border-bottom: 1px solid rgb(223, 223, 223);
+  padding-left: 25px;
+  display: flex;
+  align-items: center;
+  img {
```

Comment:
```suggestion
  align-items: center;
  
  img {
```
새로운 셀렉터가 시작될 때는 한줄 띄워주세요

Reply:
네 !